### ISSUE TITLE
Report linenumber in plugin errors

### ISSUE THREAD

**Issue by micheldebree** [nurpax/c64jasm#85]
When I develop javascript in a plugin alongside my main asm, a javascript error is reported like this example:

```
twister.asm|40 col 1| error: Plugin load failed: twister.js.js: Unexpected token '!'
```

This refers to the asm file where the js is used. For debugging I would like to see the filename of the plugin, and the line,col where the error occured. Preferably in the same error format so I can click through to the source.
My workaround is to write tests in javascript to find bugs.

**Comment by nurpax** [comment]
Agreed, I found this a bit of a pain too, and worked similarly by writing various test functions within the plugin js files.  

Definitely will look into this.

**Comment by nurpax** [comment]
I just published c64jasm 0.9.2 that includes a fix for this issue.

**Comment by micheldebree** [comment]
Thanks, seems to work like a charm!
